demo/src/main/res/layout/include_device_storage_root.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:layout_editor_absoluteY="81dp">
+
+  <TextView
```

Comment:
Worth adding also the type of storage from the user's perspective? like internal, sd card, etc..?

Reply:
I was thinking about doing a more in-depth demo but at the moment this is to get something working so that we can add to the `download-manager`.